src/Website/Resources/orderconfirmationemail.php

```diff
@@ -6,14 +6,14 @@
 //$mail->SMTPDebug = 3;                               // Enable verbose debug output
 
 $mail->isSMTP();                                      // Set mailer to use SMTP
-$mail->Host = 'mail.smtp2go.com';  // Specify main and backup SMTP servers
+$mail->Host = 'smtp.sendgrid.net';  // Specify main and backup SMTP servers
 $mail->SMTPAuth = true;                               // Enable SMTP authentication
-$mail->Username = 'orders@stuffbuysharks.xyz';                 // SMTP username
-$mail->Password = 'CS3250!!';                           // SMTP password
+$mail->Username = 'apikey';                 // SMTP username // apikey CS3250Key2
```

Comment:
we worked on getting this working with the new email server!!

Reply:
Saw it working as well. Thanks Derek